File: apps/problems/two-sum/boilerplate-full/function.cpp

#include <iostream>
#include <vector>
#include <string>
#include <algorithm>
#include <iterator>

##USER_CODE_HERE##

int main() {
    	int num1;
	std::cin >> num1;

 	int num2;
	std::cin >> num2;

    int result = twoSum(num1, num2);

    std::cout << result << std::endl;

    return 0;
}
        

File: apps/problems/two-sum/boilerplate-full/function.rs

use std::io::{self, BufRead};
use std::str::FromStr;

##USER_CODE_HERE##

fn main() {
    let stdin = io::stdin();
    let mut input = stdin.lock().lines().map(|l| l.unwrap());
    let num1: i32 = input.next().unwrap().parse().unwrap();

 let num2: i32 = input.next().unwrap().parse().unwrap();

    let result = twoSum(num1, num2);

    println!("{}", result);

}
        